_sass/_sign-the-letter.scss

```diff
@@ -0,0 +1,101 @@
+  .jumbotron{
+    margin-top: -45px;
+    padding:0;
+    position:relative;
+    overflow: hidden;
+    @include phone{
+      margin-top: 0px;
+    }
+  }
+  .jumbotron>.container {
+    padding-top:30px;
+    padding-bottom:15px;
+    // background: url("/images/campaigns/20150429-hero-bg.jpg") 0 -0 no-repeat;
+    position:relative;
+    z-index: 5;
+    background-size: cover;
+    box-shadow: 0 0 50px rgba(22,33,63,0.7);
+    @include phone{
+      background-size: 50%;
+    }
+
+  }
+  
+  .jumbotron>.container>.row{
+    background: url("/images/campaigns/20150429-hero-white.png") 0 -0 no-repeat;
+    background-size: contain;
+  }
+
+  .jumbotron .bg-blurred{
+    filter: url(blur.svg#blur); -webkit-filter: blur(3px); filter: blur(3px);
+    background: url("/images/campaigns/20150429-hero-bg.jpg") 0 -3px no-repeat;
+    position:absolute;
+    width:105%;
+    height:105%;
+    left:-5px;
```

Comment:
beacuse of the way this page is disposable, I actually don't want to break out the CSS.   There are going to be loads of campaign pages over time, so I don't want to mingle the old css with the site-wide css.